sandbox/grommet-app/src/pages/layouts/kinds/OpsRampDetail/Config.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
```

Comment:
File name should probably lower case. Naming conventions in React typically reserve the title case for JSX components and types.

This is more of a utility file.